### PR TITLE
Gameplay_Init  matched

### PR DESCRIPTION
**Pull request by shawlucas**
yeah

### CODE REVIEW

File: src/code/z_play.c [hunk 1]
```diff
@@ -260,53 +255,50 @@ void Gameplay_Init(GameState* thisx) {
         gSaveContext.environmentTime = gSaveContext.nextDayTime;
     }
 
-    if ((gSaveContext.dayTime >= 0xC001) || (gSaveContext.dayTime < 0x4555)) {
+    if (gSaveContext.dayTime >= 0xC001 || gSaveContext.dayTime < 0x4555) {
```
Comment (fig02) [inline src/code/z_play.c:258]: > 0xC000

Comment (fig02) [reply src/code/z_play.c:258]: lmfao it turned my greater than sign into a quote

**Review by fig02 — APPROVED** [review]

File: src/code/z_play.c [hunk 2]
```diff
@@ -260,53 +255,50 @@ void Gameplay_Init(GameState* thisx) {
...
+               gSaveContext.sceneSetupIndex < 4) {
+        gSaveContext.sceneSetupIndex = (gSaveContext.eventChkInf[4] & 0x100) ? 3 : 2;
     }
 
-    spawnEntrance = &gEntranceTable[gSaveContext.entranceIndex + gSaveContext.sceneSetupIndex];
-    Gameplay_SpawnScene(globalCtx, spawnEntrance->scene, spawnEntrance->spawn);
-    osSyncPrintf("\nSCENE_NO=%d COUNTER=%d\n", gSaveContext.entranceIndex, gSaveContext.sceneSetupIndex);
+    Gameplay_SpawnScene(
+        globalCtx, (&gEntranceTable[(((void)0, gSaveContext.entranceIndex)) + (((void)0, gSaveContext.sceneSetupIndex))])->scene,
+        (&gEntranceTable[(((void)0, gSaveContext.sceneSetupIndex)) + (((void)0, gSaveContext.entranceIndex))])->spawn);
```
Comment (Roman971) [inline src/code/z_play.c:296]: Any chance these lines can be made a bit cleaner like this?
```suggestion
        globalCtx, gEntranceTable[((void)0, gSaveContext.entranceIndex) + ((void)0, gSaveContext.sceneSetupIndex)].scene,
        gEntranceTable[((void)0, gSaveContext.sceneSetupIndex) + ((void)0, gSaveContext.entranceIndex)].spawn);
```

**Comment by fig02** [comment]
reminder to try out romans suggestion when you find the time

**Review by fig02 — APPROVED** [review]

**Review by Roman971 — APPROVED** [review]

**Pull request merged**